.gitignore

```diff
@@ -6,3 +6,4 @@
 /dist
 /build
 /docs/_build
+/venv
```

Comment:
Let's revert this change as it's not critical for this patch.

Reply:
Yes, that's temporary and won't go to the final commit.